src/Microsoft.ML.DnnImageFeaturizer.AlexNet/Microsoft.ML.DnnImageFeaturizer.AlexNet.csproj

```diff
@@ -16,10 +16,26 @@
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" >
       <PrivateAssets>all</PrivateAssets>
     </ProjectReference>
+
+    <PackageReference Include="Microsoft.ML.Assets" Version="$(MicrosoftMLAssetsVersions)" ExcludeAssets="all" GeneratePathProperty="true"/>
   </ItemGroup>
 
   <ItemGroup>
     <Content Include="$(RepoRoot)eng\pkg\DnnImageFeaturizer.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ModelFileCopy Include="$(PkgMicrosoft_ML_Assets)\contentFiles\any\any\AlexNetPrepOnnx\AlexNetPreprocess.onnx"
+    TargetPath="$(PackageAssetsPath)Microsoft.ML.DnnImageFeaturizer.AlexNet\tools\DnnImageModels\AlexNetPrepOnnx\AlexNetPreprocess.onnx" />
```

Comment:
(nit) these are easier to read if they are formatted like the following

![image](https://user-images.githubusercontent.com/8291187/110671916-b1057580-8194-11eb-9d1a-f8a5b1364cb1.png)

Notice that all attributes are aligned.